### PR TITLE
add primitive matrix to phonopyJob

### PR DESCRIPTION
**Pull request by caglayanoaras**

### CODE REVIEW

File: pyiron/atomistics/master/phonopy.py [hunk 1]
```diff
@@ -167,6 +168,7 @@ def _enable_phonopy(self):
                 self.phonopy = Phonopy(
                     unitcell=self._phonopy_unit_cell,
                     supercell_matrix=self._phonopy_supercell_matrix(),
+                    primitive_matrix=self.primitive_matrix,
```
Comment (jan-janssen) [inline pyiron/atomistics/master/phonopy.py:171]:
```suggestion
                    primitive_matrix=self.input[“primitive_matrix”],
```

File: pyiron/atomistics/master/phonopy.py [hunk 2]
```diff
@@ -142,6 +142,7 @@ def __init__(self, project, job_name):
         self.input["dos_mesh"] = (20, "mesh size for DOS calculation")
 
         self.phonopy = None
+        self.primitive_matrix = None
```
Comment (jan-janssen) [inline pyiron/atomistics/master/phonopy.py:145]:
```suggestion
```

**Comment by jan-janssen** [comment]
Use generic parameters to store the input parameters.

**Review by jan-janssen — APPROVED** [review]
Looks good to me, feel free to merge it once the tests passed

**Comment by jan-janssen** [comment]
@caglayanoaras Nice work and congratulations for merging your first pull request.

**Comment by caglayanoaras** [comment]
@jan-janssen Thank you!

**Pull request merged**